### DISCUSSION TITLE
v1 - General

### DISCUSSION THREAD

**Discussion by alisonjoseph** [carbon-design-system/carbon-platform#1369] [Release feedback]
👋 Hi, have you checked out our v1 public beta release? We'd love to hear any general feedback you might have about the release here. 

Thanks so much for your help! We really appreciate it 🙏

**Comment by pswiekibm** [comment]
Love the new platform.  The components page is really slick and it's nice seeing the rich content style list.  The feedback I have there is that because there are so many components, it is tough for me to know all that exists from the current list view.  The previous website had the sidebar navigation simple list that was easy to see everything and get a birds eye view.  Is this type of higher level overview still available?  If not, I'd love to have that type of view again.  It can be a bit overwhelming to scroll/page through the 210 current items when I don't know what's there.  Thank you!  Happy to discuss further with you all!

**Reply by mattrosno** [reply]
Reply: https://github.com/carbon-design-system/carbon-platform/discussions/1369#discussioncomment-3765610

**Comment by jeanservaas** [comment]
Hey @pswiekibm Thanks for the feedback and I understand where you're coming from. It is definitely a different experience from the status quo. We're also trying to avoid some of the pitfalls that Carbon for IBM Products encountered when they collapsed a large amount of assets into the left hand nav. 

I'm not sure if this will help but you can view the list of assets in each library as well... so If you want to see all of the assets in the current carbon left hand nav, you can look under Assets in the Carbon React library.

https://next.carbondesignsystem.com/libraries/carbon-react/latest/assets

Agree this NEEDS to be more intuitive and we're working on making more of a guided library experience so people understand better what libraries are foundational for them.

We'd also love to work with you in the future as we continue to iterate on the experience. We were able to pair with Morgana to get some sketches before she left, which was invaluable. But we definitely need more design/ux partnerships in in the product world.

**Reply by pswiekibm** [reply]
Thanks!  Totally agree and I didn't see that page before.  I definitely don't think that it should be in the side navigation, so happy to hear that everyone is working on new solutions.  Happy to continue to collaborate with the team - you can find me on IBM Slack!
